cat-theme/src/js/main.js

```diff
@@ -0,0 +1,74 @@
+const app = (() => {
+	let body = document.querySelector('body');
+	let darkBackgroundElement = document.querySelector(".dark--background");
+	let menuHeader = document.querySelector('.menu-title');
+	let menuIcon = document.querySelector('.menu-icon');
+	let parallax = document.querySelector(".parallax");
+
+
+	menuHeader.addEventListener('click', () => body.classList.toggle('nav-active'));
+	menuIcon.addEventListener('click', () => body.classList.toggle('nav-active'));
+
+	window.addEventListener('scroll', function () {
+		debounce(scrollParallax(parallax));
+
+		debounce(changeHamburgerColor(darkBackgroundElement));
+	}, false);
+})();
+
+const debounce = fn => {
+	let timeout;
+
+	return () => {
+		if (timeout) {
+			window.cancelAnimationFrame(timeout);
+		}
+
+		timeout = window.requestAnimationFrame(function () {
+			fn.apply(this, arguments);
+		});
+	}
+};
+
+const isInViewport = element => {
+	const bounding = element.getBoundingClientRect();
+
+	if (bounding.top >= 0 || bounding.bottom >= 0) {
+		return true;
+
+	} else {
+		return false;
+	}
```

Comment:
Instead of this entire `if`/`else`, can we just do `return bounding.top >= 0 || bounding.bottom >= 0;`